R/tag_query.R

```diff
@@ -631,14 +631,21 @@ tagQuery_ <- function(
             tagQueryFindClosest(selected_, cssSelector)
           )
         },
-        #' ### Custom filter
+        #' ### Filter
         #'
+        #' * `$matches(fn)`: For each of the selected tags, return `TRUE` if
+        #'  `fn(el)` returns `TRUE`. In addition to an R function with two
+        #'   arguments (the selected tag `x` and the index `i`), `fn` may also
+        #'   be a valid CSS selector.
+        matches = function(fn) {
```

Comment:
I feels a bit weird to have the 1st method under a "Filter" section be `matches` and the 2nd one be `filter`